src/mongo.constants.ts

```diff
@@ -5,3 +5,5 @@ export const MONGO_MODULE_OPTIONS = 'MongoModuleOptions'
 
 export const DEFAULT_MONGO_CONNECTION_NAME = 'DefaultMongo'
 export const DEFAULT_MONGO_CLIENT_OPTIONS: MongoClientOptions = { useNewUrlParser: true }
+
+export const DEFAULT_MONGO_COLLECTION_NAME = 'DefaultCollection'
```

Comment:
Is there a situation where you'd want a "default" collection?  It seems like something you'd always want to specify explicitly.

Reply:
You're right here. There shouldn't be one. The collection name should be a mandatory parameter. I've fixed this.